### ISSUE TITLE
Location not updating in background again

### ISSUE THREAD

**Issue by mrdanleey** [Myzel394/locus#93]
I installed the newest version 0.13.0 (both) and again it isn't updating location in the background

**Comment by mrdanleey** [comment]
Update:

Updating only works when notification is active

**Comment by Myzel394** [comment]
Can you share me your logs?

**Comment by mrdanleey** [comment]
[29072023_204747_PM_all_[281].zip](https://github.com/Myzel394/locus/files/12207780/29072023_204747_PM_all_.281.zip)

Here are the logs from the phone of my wife. On my phone I cannot get the ligs as it give a error

**Comment by Myzel394** [comment]
Thanks for the logs, unfortunately I can't extract them. I'm currently investigating this issue, but it seems as if it's a problem with location fetching itself, as background processing is working flawlessly on my devices. In the far future I'll probably want to add a custom implementation for updating the location in the background, this should hopefully eliminate this issue at the very end once and for all. 

Can you verify Locus can access your location by doing a location check and telling me whether it was successful? You may also want to wait until 0.14.0 (should come out in the next few days), which will fix some bugs here and there, maybe this will also fix this bug for you.

**Comment by mrdanleey** [comment]
Yes, Locus can fetch my location. I'll wait for 0.14 version. Thanks!

**Comment by Myzel394** [comment]
Sorry for the long wait @mrdanleey. Originally, I wanted to wait for F-Droid to integrate Locus to avoid any problems with it; seems as if it'll take some time for that though... I've published v0.14.0 now - can you confirm if background location fetching works now?

**Comment by Myzel394** [comment]
@mrdanleey did the new version fix the issue for you?

**Comment by mrdanleey** [comment]
On my phone it works now, on the other two phones not. It eon't load the location on the background. Maybe it's a android problem? (Android 10 & 11)

**Comment by Myzel394** [comment]
What Android version is your phone running?

**Comment by mrdanleey** [comment]
Android 13.

Trying new version now and can't get it working in the background too. Weird. I have the same settings as the previous version but can't get it to work in the background.

In the previous version it took a little while to work in the background but now nothing works.

**Comment by Myzel394** [comment]
Maybe you are using a slottery machine, hence why it's behaving so randomly :D

Unfortunately, I'm not able to reproduce this, as everything is running fine on my phone with Android 13 :/. Just to make sure, you are using version `0.14.2 (FLOSS)`? Can you try creating a task, closing the app and then waiting for at least one hour? If it doesn't update, can you try triggering a manual update?

After that, please send me your log files, I'll take a look at them
